pathplannerlib/src/main/native/cpp/pathplanner/lib/auto/BaseAutoBuilder.cpp

```diff
@@ -77,66 +72,64 @@ frc2::CommandPtr BaseAutoBuilder::wrappedEventCommand(
 	return std::move(wrapped).ToPtr();
 }
 
-frc2::CommandPtr BaseAutoBuilder::stopEventGroup(
+frc2::CommandPtr BaseAutoBuilder::getStopEventCommands(
 		PathPlannerTrajectory::StopEvent stopEvent) {
-	std::vector < std::unique_ptr < frc2::Command >> eventCommands;
-
+	std::vector < frc2::CommandPtr > eventCommands;
 	for (size_t i =
 			(stopEvent.executionBehavior
 					== PathPlannerTrajectory::StopEvent::ExecutionBehavior::PARALLEL_DEADLINE ?
 					1 : 0); i < stopEvent.names.size(); i++) {
 		std::string name = stopEvent.names[i];
 		if (m_eventMap.find(name) != m_eventMap.end()) {
 			eventCommands.emplace_back(
-					wrappedEventCommand(m_eventMap.at(name)).Unwrap());
+					wrappedEventCommand(m_eventMap.at(name)));
 		}
 	}
 
-	frc2::CommandPtr events = frc2::InstantCommand().ToPtr();
 	if (stopEvent.executionBehavior
 			== PathPlannerTrajectory::StopEvent::ExecutionBehavior::SEQUENTIAL) {
-		events = frc2::SequentialCommandGroup(std::move(eventCommands)).ToPtr();
+		return frc2::cmd::Sequence(std::move(eventCommands));
 	} else if (stopEvent.executionBehavior
 			== PathPlannerTrajectory::StopEvent::ExecutionBehavior::PARALLEL) {
-		events = frc2::ParallelCommandGroup(std::move(eventCommands)).ToPtr();
+		return frc2::cmd::Parallel(std::move(eventCommands));
 	} else if (stopEvent.executionBehavior
 			== PathPlannerTrajectory::StopEvent::ExecutionBehavior::PARALLEL_DEADLINE) {
-		frc2::CommandPtr deadline = frc2::InstantCommand().ToPtr();
-		if (m_eventMap.find(stopEvent.names[0]) != m_eventMap.end()) {
-			deadline = wrappedEventCommand(m_eventMap.at(stopEvent.names[0]));
-		}
-		events = frc2::ParallelDeadlineGroup(std::move(deadline).Unwrap(),
-				std::move(eventCommands)).ToPtr();
+		frc2::CommandPtr *deadline = &eventCommands.front();
+		eventCommands.erase(eventCommands.begin());
+		return frc2::cmd::Deadline(std::move(*deadline),
```

Comment:
Please fix this 😄

Reply:
```C++
frc2::CommandPtr deadline = frc2::cmd::None();
if (m_eventMap.find(stopEvent.names[0]) != m_eventMap.end()) {
	deadline = wrappedEventCommand(m_eventMap.at(stopEvent.names[0]));
}
return frc2::cmd::Deadline(std::move(deadline), std::move(eventCommands));
```

The deadline command is already left out of the `eventCommands` list so you don't need to do that again.

Also, the final else if block in stopEventGroup has frc2::Command as the type in the vector instead of frc2::CommadPtr, this will fail to build

Reply:
Ahh, you right! Thanks